janusgraph-server/src/main/java/org/janusgraph/graphdb/grpc/schema/SchemaManagerProvider.java

```diff
@@ -60,4 +63,28 @@ public List<EdgeLabel> getEdgeLabels() {
             .stream(management.getRelationTypes(org.janusgraph.core.EdgeLabel.class).spliterator(), false)
             .map(GrpcUtils::createEdgeLabelProto).collect(Collectors.toList());
     }
+
+    public VertexCompositeGraphIndex getVertexCompositeGraphIndexByName(String indexName) {
+        JanusGraphIndex graphIndex = management.getGraphIndex(indexName);
+        if (graphIndex == null) {
+            return null;
+        }
+        if (!Vertex.class.isAssignableFrom(graphIndex.getIndexedElement())) {
+            return null;
+        }
+
+        if (!graphIndex.isCompositeIndex()) {
+            return null; //error?
```

Comment:
I also think that an error would make more sense than just returning `null` which could also mean that no index was found. If no index was found, a user might try next to create it which will fail if it already exists.
This should also be tested.